epistemix_platform/infrastructure/tests/ecr/test_simulation_runner_repository_template.py

```
@@ -572,3 +570,135 @@ def test_ecr_scan_event_rule_not_present(self, ecr_template: dict[str, Any]):
         assert (
             "ECRScanEventRule" not in resources
         ), "ECRScanEventRule resource should not exist after SNS notification removal"
+
+
+    @pytest.mark.integration
+    def test_template_passes_cfn_lint(self, ecr_template_path: str, cfnlint_config_path: str):
+        """Test that the ECR template passes cfn-lint validation."""
+        import subprocess
+
+        result = subprocess.run(
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)